Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:20.04
 ARG USER_UID
 RUN apt-get update -q && apt-get install -y git curl gnupg jq build-essential gawk zip 
-RUN curl -s https://dist.ipfs.io/go-ipfs/v0.8.0/go-ipfs_v0.8.0_linux-amd64.tar.gz | tar vzx -C /usr/local/bin/ go-ipfs/ipfs --strip-components=1
+RUN curl -s https://dist.ipfs.io/go-ipfs/v0.9.1/go-ipfs_v0.9.1_linux-amd64.tar.gz | tar vzx -C /usr/local/bin/ go-ipfs/ipfs --strip-components=1
```

Comment:
Similarly, what's the process for updating this?
Can we have a URL like https://dist.ipfs.io/go-ipfs/latest-full-release/go-ipfs_linux-amd64.tar.gz

Reply:
Good catch. Simplified in 7d222b5 which makes sure HTTP client here uses same version as `GO_IPFS_VER` – just in case we have breaking change  in any of used APIs.